### ISSUE TITLE
Failed to create vpool on distributed filesystemd

### ISSUE THREAD

**Issue by grimpy** [openvstorage/framework#1011]
```
- ERROR - Task ovs.storagerouter.add_vpool[ef36fbf7-e3e9-497e-8759-e84b6886af8a] raised unexpected: TypeError('Python argument types in\n    ClusterNodeConfig.__init__(ClusterNodeConfig)\ndid not match C++ signature:\n    __init__(_object*, volumedriverfs::NodeId vrouter_id, std::string host, volumedriverfs::MessagePort message_port, volumedriverfs::XmlRpcPort xmlrpc_port, volumedriverfs::FailoverCachePort failovercache_port)',)
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/lib/storagerouter.py", line 555, in add_vpool
    node_configs.append(ClusterNodeConfig(**storagedriver.cluster_node_config))
Exception: Python argument types in
    ClusterNodeConfig.__init__(ClusterNodeConfig)
did not match C++ signature:
    __init__(_object*, volumedriverfs::NodeId vrouter_id, std::string host, volumedriverfs::MessagePort message_port, volumedriverfs::XmlRpcPort xmlrpc_port, volumedriverfs::FailoverCachePort failovercache_port)
```

**Comment by khenderick** [comment]
Fixed by #1016, packaged in openvstorage-2.7.4-rev.4113.d04e948

**Comment by kinvaris** [comment]
Distributed filesystem vPools have been removed by the engineering team as it is not supported anymore.
